.eslintrc

```diff
@@ -9,32 +9,75 @@
   },
   "parser": "babel-eslint",
   "plugins": [
-    "react",
-    "babel"
+    "react"
   ],
   "rules": {
-    "constructor-super": 2,
-    "comma-dangle": 0,
-    "eqeqeq": [2, "allow-null"],
-    "id-length": 0,
-    "indent": [2, 2, { "SwitchCase": 1, "VariableDeclarator": 1 }],
-    "no-eq-null": 0,
-    "one-var": [2, { "initialized": "never" }],
-    "prefer-const": 0,
-    "no-param-reassign": 0,
-    "no-this-before-super": 2,
-    "babel/object-shorthand": 2,
+    "max-len": "off",
+    "no-plusplus": "off",
+    "prefer-const": "off",
+    "one-var": "off",
+    "no-param-reassign": "off",
+    "no-mixed-operators": ["error", {
+      "groups": [
+        ["&", "|", "^", "~", "<<", ">>", ">>>"],
+        ["==", "!=", "===", "!==", ">", ">=", "<", "<="],
+        ["&&", "||"],
+        ["in", "instanceof"]
+      ],
+      "allowSamePrecedence": false
+    }],
+    "class-methods-use-this": "off",
```

Comment:
i think the 4catalyzer config already handles many of these? it'd save some extra overrides on your end maybe

Reply:
true, i didn't think about using that...